### ISSUE TITLE
[Bug]ANR in org.mozilla.fenix.nightly after changing the devices language and input method

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8459]
## Prerequisites
Have a clean profile or a fresh install of Fenix.
Change the devices Language and input method to German (Deutsch).

## Steps to reproduce
1. Launch Fenix
2. Navigate to the Add-ons manager and install uBlock
3. Open the Settings section
4. Disable / Enable uBlock 
5. Open again the uBlock's Settings section
6. Repeat a couple of times steps 4 & 5

### Expected behavior
Ublock's Settings section is properly displayed each time.

### Actual behavior
Settings section freezes, resulting ANR.
Seems that the Google Assistant is automatically launched.

### Device information

* Android device: Google Pixel 3a (Android 9)
* Fenix version: Nightly Build #20470607 from 2/16

### Notes

► Video
![20200217-123200](https://user-images.githubusercontent.com/51314259/74652215-34b98200-518e-11ea-9180-fcac36fcdf98.gif)

► Log
```kotlin
2020-02-17 12:29:04.126 1172-1197/? E/ActivityManager: ANR in org.mozilla.fenix.nightly (org.mozilla.fenix.nightly/.App)
    PID: 31281
    Reason: Input dispatching timed out (Waiting to send key event because the focused window has not finished processing all of the input events that were previously delivered to it.  Outbound queue length: 0.  Wait queue length: 1.)
    Load: 2.63 / 1.95 / 1.72
    CPU usage from 41287ms to 0ms ago (2020-02-17 12:28:19.300 to 2020-02-17 12:29:00.587):
      12% 31281/org.mozilla.fenix.nightly: 10% user + 2.1% kernel / faults: 71929 minor 7 major
      6.5% 32332/com.android.vending: 5.7% user + 0.7% kernel / faults: 12973 minor 1 major
      4.2% 1172/system_server: 1.6% user + 2.5% kernel / faults: 1332 minor
      2.8% 1554/com.android.systemui: 2% user + 0.7% kernel / faults: 40 minor
      2.5% 573/surfaceflinger: 1.5% user + 0.9% kernel / faults: 2 minor
      1.3% 22404/kworker/u16:8: 0% user + 1.3% kernel
      1.1% 30000/kworker/u16:2: 0% user + 1.1% kernel
      1% 32346/com.google.android.apps.maps: 0.9% user + 0.1% kernel / faults: 3532 minor 15 major
      1% 775/android.hardware.wifi@1.0-service: 0.7% user + 0.3% kernel / faults: 317 minor
      1% 575/android.hardware.graphics.composer@2.2-service: 0.4% user + 0.5% kernel / faults: 5 minor
      0.9% 27371/com.google.android.googlequicksearchbox:search: 0.7% user + 0.1% kernel / faults: 506 minor
      0.7% 15277/com.google.android.gms.persistent: 0.5% user + 0.2% kernel / faults: 335 minor
      0.6% 400/crtc_commit:97: 0% user + 0.6% kernel
      0.6% 921/adbd: 0% user + 0.5% kernel / faults: 28735 minor
      0.6% 408/mmc-cmdqd/0: 0% user + 0.6% kernel
      0.5% 31466/org.mozilla.fenix.nightly:tab: 0.3% user + 0.1% kernel / faults: 255 minor
      0.4% 2058/irq/144-1436400: 0% user + 0.4% kernel
      0.4% 22419/kworker/u16:12: 0% user + 0.4% kernel
      0.3% 543/sugov:0: 0% user + 0.3% kernel
      0.2% 1838/cds_mc_thread: 0% user + 0.2% kernel
      0.2% 23270/kworker/5:1: 0% user + 0.2% kernel
      0.2% 257/kgsl_worker_thr: 0% user + 0.2% kernel
      0.2% 786/msm_irqbalance: 0% user + 0.1% kernel
      0.1% 30288/kworker/3:1: 0% user + 0.1% kernel
      0.1% 47/ksoftirqd/4: 0% user + 0.1% kernel
      0.1% 788/lmkd: 0% user + 0.1% kernel
      0.1% 7/rcu_preempt: 0% user + 0.1% kernel
      0.1% 56/ksoftirqd/5: 0% user + 0.1% kernel
      0.1% 30597/com.google.android.apps.messaging: 0% user + 0% kernel / faults: 20 minor
      0.1% 536/logd: 0% user + 0% kernel
      0.1% 24421/com.android.chrome:sandboxed_process0: 0% user + 0% kernel
      0.1% 24607/com.android.chrome:sandboxed_process0: 0% user + 0% kernel
      0% 50/rcuop/4: 0% user + 0% kernel
      0% 401/crtc_event:97: 0% user + 0% kernel
      0% 30610/com.google.android.dialer: 0% user + 0% kernel / faults: 28 minor
      0% 30835/com.google.android.play.games: 0% user + 0% kernel / faults: 27 minor
      0% 38/ksoftirqd/3: 0% user + 0% kernel
      0% 766/android.hardware.memtrack@1.0-service: 0% user + 0% kernel
      0% 843/statsd: 0% user + 0% kernel
      0% 2110/com.google.process.gservices: 0% user + 0% kernel / faults: 31 minor 1 major
      0% 2987/com.google.modemservice: 0% user + 0% kernel / faults: 2 minor
      0% 30275/kworker/2:0: 0% user + 0% kernel
      0% 32402/com.google.process.gapps: 0% user + 0% kernel / faults: 220 minor
      0% 3/ksoftirqd/0: 0% user + 0% kernel
      0% 10/rcuop/0: 0% user + 0% kernel
      0% 29/ksoftirqd/2: 0% user + 0% kernel
      0% 32/rcuop/2: 0% user + 0% kernel
      0% 65/ksoftirqd/6: 0% user + 0% kernel
      0% 544/sugov:6: 0% user + 0% kernel
      0% 655/netd: 0% user + 0% kernel
      0% 687/wlan_logging_th: 0% user + 0% kernel
      0% 755/android.hardware.camera.provider@2.4-service_64: 0% user + 0% kernel
      0% 787/audioserver: 0% user + 0% kernel
      0% 845/wificond: 0% user + 0% kernel / faults: 6 minor
      0% 851/qcrild: 0% user + 0% kernel / faults: 5 minor
      0% 1746/com.android.phone: 0% user + 0% kernel / faults: 9 minor
      0% 22412/kworker/4:2: 0% user + 0% kernel
      0% 24338/com.hecorat.screenrecorder.free: 0% user + 0% kernel / faults: 6 minor
      0% 29468/kworker/0:1: 0% user + 0% kernel
      0% 30101/irq/272-synapti: 0% user + 0% kernel
      0% 30348/kworker/u17:4: 0% user + 0% kernel
      0% 31790/com.google.android.apps.ph
2020-02-17 12:29:04.135 1172-1197/? W/ActivityManager: Activity pause timeout for ActivityRecord{249c842 u0 org.mozilla.fenix.nightly/.App t65}
2020-02-17 12:29:04.136 2145-2145/? I/GsaVoiceInteractionSrv: Handling ACTION_START_DSP_HOTWORD
2020-02-17 12:29:04.154 1554-1554/? I/vol.Events: writeEvent dismiss_dialog volume_controller
2020-02-17 12:29:04.154 1554-1554/? V/StatusBar: mStatusBarWindow: com.android.systemui.statusbar.phone.StatusBarWindowView{dc174db V.ED..... ........ 0,0-1080,66} canPanelBeCollapsed(): false
2020-02-17 12:29:04.154 1554-1554/? I/vol.Events: writeEvent dismiss_dialog volume_controller
2020-02-17 12:29:04.154 1554-1554/? V/StatusBar: mStatusBarWindow: com.android.systemui.statusbar.phone.StatusBarWindowView{dc174db V.ED..... ........ 0,0-1080,66} canPanelBeCollapsed(): false
2020-02-17 12:29:04.154 1554-1554/? I/vol.Events: writeEvent dismiss_dialog volume_controller
2020-02-17 12:29:04.155 1554-1554/? V/StatusBar: mStatusBarWindow: com.android.systemui.statusbar.phone.StatusBarWindowView{dc174db V.ED..... ........ 0,0-1080,66} canPanelBeCollapsed(): false
2020-02-17 12:29:04.158 752-9254/? D/sound_trigger_hw: stdev_start_recognition:[8] Enter
2020-02-17 12:29:04.158 752-9254/? D/platform_realtek: platform_start_recognition: hotword
2020-02-17 12:29:04.158 752-9254/? D/sound_trigger_hw: stdev_start_recognition:[8] Exit
2020-02-17 12:29:04.159 1172-8928/? I/SoundTriggerHelper: startRecognition successful.
```

**Comment by psymoon** [comment]
@AndiAJ I can't seem to reproduce this. Is this happening on pixel 3a only? Does this happen every time? The log suggests that this may be related to google assistant - does it happen when google assistant doesn't pop up?

I don't think this is a S1 bug.

**Comment by csadilek** [comment]
I can reproduce this reliably only on the Pixel 3A. The steps can be simplified though. It's enough to just hit settings and back repeatedly. Let's investigate once we have all store and pending GV patches in which could fix this.

**Comment by AndiAJ** [comment]
Indeed @csadilek , I've just re-checked this on the latest **`Nightly Build #20520607 from 2/21`** using a Google Pixel 3a (Android 9)

@psymoon It looks like the Google assistant is launched after tapping the home button.

► Video
![20200221-172046](https://user-images.githubusercontent.com/51314259/75048144-b7b14400-54d0-11ea-9557-893e0a71bebc.gif)

**Comment by csadilek** [comment]
@AndiAJ with the latest fixes, I can no longer reproduce on my Pixel 3a (whereas before I got the ANR pretty quickly.) Can you verify?

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20550606 from 02/24`** using the following devices:
• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Google Pixel 3 (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Video from Google Pixel 3a
![20200224-123217](https://user-images.githubusercontent.com/51314259/75145675-14477580-5702-11ea-89ea-a940568b1241.gif)
